contrib/opentimelineio_contrib/application_plugins/tests/test_rv_reader.py

```diff
@@ -0,0 +1,246 @@
+#
+# Copyright Contributors to the OpenTimelineIO project
+#
+# Licensed under the Apache License, Version 2.0 (the "Apache License")
+# with the following modification; you may not use this file except in
+# compliance with the Apache License and the following modification to it:
+# Section 6. Trademarks. is deleted and replaced with:
+#
+# 6. Trademarks. This License does not grant permission to use the trade
+#    names, trademarks, service marks, or product names of the Licensor
+#    and its affiliates, except as required to comply with Section 4(c) of
+#    the License and to reproduce the content of the NOTICE file.
+#
+# You may obtain a copy of the Apache License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the Apache License with the above modification is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied. See the Apache License for the specific
+# language governing permissions and limitations under the Apache License.
+#
+
+"""Unit tests for the rv reader plugin"""
+
+# Please note that the test is geared towards linux install as I don't have
+# a Windows or Mac available to me.
+
+import os
+import sys
+import ast
+import zipfile
+import tempfile
+import unittest
+import shutil
+import shlex
+import time
+from subprocess import call, Popen, PIPE
+
+import opentimelineio as otio
+
+RV_OTIO_READER_NAME = 'Example OTIO Reader'
+RV_OTIO_READER_VERSION = '1.0'
+
+RV_ROOT_DIR = os.getenv('OTIO_RV_ROOT_DIR', '')
+RV_BIN_DIR = os.path.join(RV_ROOT_DIR, 'bin')
+
+RV_OTIO_READER_DIR = os.path.join(
+    '..',
+    'rv',
+    'example_otio_reader'
+)
+
+# Need this for connecting to RV
+RV_PYTHON_DIR = os.path.join(RV_ROOT_DIR, 'plugins', 'Python')
+if RV_PYTHON_DIR not in sys.path:
+    sys.path.append(RV_PYTHON_DIR)
+
+import rv_tests_network     # noqa: E402
```

Comment:
Cool! However I was curious about this import as I don't see it in my rv/plugins/Python dir. Is it coming from somewhere else I might not be aware of?

Reply:
Oh, I'll check if it's in an older release of RV as well. I think the same class I used is available under src in another module

Reply:
@avrata, there's an `RvComminnicator` class in `src/python/network/rvNetwork.py` but the network folder doesn't contain an `__init__.py`. I think I'll do what others seem to do and add the class to our test files in stead. 
Agree?

Reply:
Seems fine with me, however I can't speak for any of the license issues or whatnot with including this code here. Someone closer to the project might need to chime in.

Reply:
I believe it's what "everyone" does, but would be great to have a thumbs up from someone at RV HQ. The file itself doesn't contain any license.
Should I push the change and revert if we get the thumbs down?

Reply:
I'm checking with the RV leadership on if this is an acceptable redistribution.

Would you prefer that RV just ships with an importable package that you can point at?  Seems like a low bar here, but if you need to support already distributed RV versions, I can understand.

Reply:
For future situations like this it would be great to be able to import directly from RV so we don't get out of sync with any changes. 
It would be nice however to be able to support older versions, so let's see what legal says and deal with it accordingly.
Another option could be to use the imp module on he current rvNetwork file I guess.